apps/examples/testcase/le_tc/network/tc_net_setsockopt.c

```diff
@@ -682,7 +682,7 @@ int net_setsockopt_main(void)
 	//tc_net_setsockopt_ipproto_ip_ip_pktinfo_p(fd);
 	tc_net_setsockopt_keepalive_p(fd);
 	//tc_net_setsockopt_rcvbuf_p(fd);
-	tc_net_setsockopt_reuseport_p(fd);
+	//tc_net_setsockopt_reuseport_p(fd);
```

Comment:
This will cause compilation warnig about "set but not used"

Reply:
I commented out that functions.